.agents/run-parallel-claude.sh

```
@@ -0,0 +1,349 @@
+    fi
+}
+
+# Parse command line arguments
+# Default to first phase by priority if no argument provided
+DEFAULT_PHASE=$(yq eval '.phases | to_entries | sort_by(.value.priority) | .[0].key' "$TODOS_YAML")
+PHASE="${1:-$DEFAULT_PHASE}"
+
+case "$PHASE" in
+    "all")
+        echo -e "${BLUE}🚀 STARTING ALL PARALLEL SESSIONS${NC}"
+        local total_todos=$(yq eval '.todos | map(select((.status // "active") != "cancelled")) | length' "$TODOS_YAML")
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)